docs/transparent-validation/sd-wire/usage-validation.md

```diff
@@ -2,109 +2,190 @@
+        ```
+        cmake ..
+        make
+        ```
+
+5. In the above-described directory (`sd-mux/build`) run the following command to 
+    build binary:
+    ```
+    sudo make install
+    ```
+    Note, that the above-described command installs binary into '/usr/local/bin'.
+    If you want to install files in directory rather than the default one add an 
+    argument to cmake command: 
+    ```
+    cmake -DCMAKE_INSTALL_PREFIX=/usr .. 
+    ```
+    Then it is obligatory to run again the following commands:
+    ```
+    make
+    make install
+    ```
 
 ## First use
 
-1. Run in the terminal:
-```
-dmesg -w
-```
-2. Connect the SDWire to your machine using micro-USB-->USB cable.
-3. Your `dmesg` output should looks like this:
-```
-(...)
-[73278.307591] usb-storage 3-1.1:1.0: USB Mass Storage device detected
-[73278.307823] scsi host6: usb-storage 3-1.1:1.0
-[73278.384925] usb 3-1.2: new full-speed USB device number 45 using xhci_hcd
-[73278.492025] usb 3-1.2: New USB device found, idVendor=0403, idProduct=6015, bcdDevice=10.00
-[73278.492027] usb 3-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
-[73278.492028] usb 3-1.2: Product: FT200X USB I2C
-[73278.492029] usb 3-1.2: Manufacturer: FTDI
-[73278.492030] usb 3-1.2: SerialNumber: DB007V7V
-(...)
-```
-
-4. Check if SDWire is configured:
-```
-sudo  sd-mux-ctrl --list
-```
-If output looks like below, it means that SDWire is configured and ready to use.
-From now on, you can go to ["Everyday use"](#everyday-use) section.
-```
-Number of FTDI devices found: 1
-Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
-```
-Otherwise, if output shows no devices (like in the example below):
-```
-Number of FTDI devices found: 0
-```
-you have to configure SDWire:
-```
-sudo sd-mux-ctrl --device-serial=DB007V7V --vendor=0x0403 --product=0x6015 --device-type=sd-wire --set-serial=sd-wire_11
-```
-where:
-```
---device-serial=<SerialNumber> (from dmesg output)
-
---vendor=0x<idVendor> (from dmesg output)
-
---product=0x<idProduct> (from dmesg output)
-
---set-serial=<New serial device>
-```
-
-5. Check if SDWire is properly configured:
-```
-sudo  sd-mux-ctrl --list
-```
-should output:
-```
-Number of FTDI devices found: 1
-Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
-```
-
-## Everyday use
-
-### Typical scenario
+Attention! The following procedure should not be performed only before the 
+first use of the device. It should also be done before selling it to another 
+user. 
+
+Hardware requirements:
+
+* SDWire
+* SD card
+* DUT (Device Under Test) bootable from SD card (RPI, Orange PI etc.)
+* DUT power supply
+* Micro-USB-->USB cable
+
+To perform first use procedure reproduce the following steps:
+
+1. Prepare environment in accordance with this [section](#environment-preparation).
+
+2. Insert SD card to the SDWire.
+
+3. Put SDWire into the DUT (Device Under Test) and connect it to the TS with 
```

Comment:
@sulewskiprzemyslaw TS in other words means personal computer or similar? Maybe we should add example in the parentheses: `(for example: local PC)`

Reply:
if not - this is confusing and needs more details

Reply:
I added a description, what TS is, to hardware requirements.